src/libstd/lib.rs

```diff
@@ -422,6 +423,11 @@ pub use core_collections::vec;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use std_unicode::char;
 
+// Reexport the start module on platforms that provide it
+#[unstable(feature = "start_fn", issue="0")]
+#[cfg(target_os = "redox")]
+pub use sys::start::*;
```

Comment:
#### Hygiene error! In order to generate executables, we have to have a naked `_start` function at the libstd root.